google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITGrpcTest.java

```diff
@@ -156,4 +188,53 @@ public void deleteHmacKey() {
     storageFixture.getInstance().updateHmacKeyState(hmacKey.getMetadata(), HmacKeyState.INACTIVE);
     storageFixture.getInstance().deleteHmacKey(hmacKey.getMetadata());
   }
+
+  @Test
+  public void object_writeGetRead() {
+    Storage s = storageFixture.getInstance();
+    BlobInfo info = BlobInfo.newBuilder(bucketFixture.getBucketInfo(), "writeGetRead").build();
+    byte[] content = "hello, world".getBytes(StandardCharsets.UTF_8);
+    s.create(info, content, BlobTargetOption.doesNotExist());
+
+    Blob blob = s.get(info.getBlobId());
+
+    byte[] actualContent = blob.getContent();
+    assertThat(actualContent).isEqualTo(content);
+  }
+
+  @Test
+  public void objectCreate_storage_create() {
```

Comment:
do you want to create a consistent prefix string for:
`objectWriteStorageCreate()`
`objectWriteStorageCreateStream()

Reply:
The intent here was to try and communicate "testing objectCreate through Storage#create"

objectCreate was due to following the fact the current methods are named create.

We can change to `objectWrite` while keeping the other hints if that sounds good to you.

Reply:
Any objection to splitting uploads / downloads IT into separate test files? They're the two main operations we are going to have many IT for.

Reply:
I've been doing that for most of the more detailed read/write tests of the new channels, these tests are mainly more of a "are things plumbed up in GrpcStorageImpl". I think, after the IT refactoring is more settled we can decided if we want to move these.

Reply:
Can we go with your original suggestion of using `objectWrite` while keeping the other hints, which I assume is `storage_create` to qualify that this is for `storage.create()`?

Reply:
will do